legal-api/src/legal_api/reports/report.py

```diff
@@ -679,6 +679,31 @@ def _format_alteration_data(self, filing):
             if new_legal_type else None
 
     def _format_amalgamation_data(self, filing):
+
+        filings = Filing.get_filings_by_types(self._business.id, ['amalgamationApplication'])
+        amalgamated_businesses = []
+
+        if filings:
+            filing['business']['amalgamatedEntity'] = True
+            amalgamation = self._business.amalgamation.one_or_none()
+            amalgamating_businesses = amalgamation.amalgamating_businesses.all()
+
+            for amalgamating_business in amalgamating_businesses:
+                if ting_business := Business.find_by_internal_id(amalgamating_business.business_id):
```

Comment:
use identifier in json to get ting business. check if foreign first (else it will get errored out saying business not found if its a foreign business)